cookiecutter/hooks.py

```diff
@@ -106,23 +105,25 @@ def run_script_with_context(script_path, cwd, context):
         mode='wb',
         suffix=extension
     ) as temp:
-        output = Template(contents).render(**context)
+        template = env.from_string(contents)
```

Comment:
Could we just instantiate `StrictEnvironment` here, instead of passing the env through all these function calls?